app/models/conversation.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Conversation < ApplicationRecord
+  acts_as_paranoid
+  belongs_to :user
+  belongs_to :message
```

Comment:
Ale, cómo va? a mi no me cierran mucho las relaciones de la base.. entiendo que los usuarios tienen muchos mensajes a través de una conversación, en ese caso, la conversación tiene muchos mensajes, no uno solo, no? y belongs_to: message hace referencia a q la conversación tiene un solo mensaje asociado.. además de eso, la conversación es entre dos usuarios, no sería has_many :users?

Reply:
Tal como dice @gervic08 , en este caso sería:
```suggestion
 has_many :messages, dependent: destroy
```